localstack/services/apigateway/apigateway_listener.py

```diff
@@ -853,7 +855,8 @@ def invoke_rest_api_integration_backend(invocation_context: ApiInvocationContext
 
     elif integration_type == "MOCK":
         # return empty response - details filled in via responseParameters above...
-        return requests_response({})
+        invocation_context.response = requests_response({})
+        return apply_integration_response_template(invocation_context)
```

Comment:
Can we add a small test to cover this new functionality - we could also extend one of the existing tests, e.g.: https://github.com/localstack/localstack/blob/4e5326605e78889fa9f81b08d97932828859e803/tests/integration/test_api_gateway.py#L1432-L1453

Reply:
Yes, can be done! I have tests in the PRO version https://github.com/localstack/localstack-ext/pull/387